src/guppy.ts

```diff
@@ -85,13 +90,10 @@ export function createInstance(config: Config) {
    * @param uuid - Id of raster
    * @param body
    */
-  async function getRasterData(
-    uuid: string,
-    body: RasterBody
-  ): Promise<number[][]> {
+  async function getRasterData(uuid: string, body: RasterBody) {
     const url = `${config.url}/rasters/${uuid}/data`;
-    const result = await axios.post(url, body);
-    return result.data.data;
+    const result = await axios.post<RasterDataResult>(url, body);
+    if (result.status === 200) return result.data.data;
```

Comment:
what if `else` ?

Reply:
Normaal is dat throw error door axios

Reply:
Omwille van async/await? Ik dacht dat de error zou ingeslikt worden, maar als dat niet zo is 👍